docs/pages/installation/linux.mdx

```diff
@@ -0,0 +1,489 @@
+---
+title: Installing Teleport on Linux
+description: How to install Teleport on Linux using our package repositories, TAR archives, or our one-line installation script for various distributions.
+---
+
+Teleport maintains DEB and RPM package repositories for different operating
+systems, platforms, and Teleport versions. A server that installs Teleport from
+a DEB or RPM package must have systemd installed. You can also download TAR
+archives containing Teleport binaries. 
+
+## Operating system support
+| Operating System | `teleport` Daemon | `tctl` Admin Tool | `tsh` and Teleport Connect User Clients [2] | Web UI (via the browser) | `tbot` Daemon |
+| - | - | - | - | - | - |
+| Linux v2.6.23+ (RHEL/CentOS 7+, Rocky Linux 8+, Amazon Linux 2+, Amazon Linux 2023+, Ubuntu 16.04+, Debian 9+, SLES 12 SP 5+, and SLES 15 SP 5+) \[3] | yes | yes | yes | yes | yes |
```

Comment:
FYI I made a small change to this table on master that should be incorporated here. https://github.com/gravitational/teleport/pull/56285